clients/unshaded/src/main/java/tachyon/client/file/FileOutStream.java

```diff
@@ -270,4 +295,15 @@ private void handleCacheWriteException(IOException ioe) throws IOException {
       mCurrentBlockOutStream.cancel();
     }
   }
+
+  private FileInfo getFileInfo(long fileId) throws IOException {
```

Comment:
I don't think this method should take `fileId` right? Since only `mFileId` should be used?